src/screens/lock/hooks/useLockScreen.ts

```diff
@@ -51,6 +52,8 @@ export const useLockScreenProps = (route?: RouteProp): LockViewProps => {
     void getData(StorageKeys.BiometryActivated).then(async activated => {
       if (!activated) return
 
+      await hideSplashScreen()
```

Comment:
My understanding is that SplashScreen would never be hiden whe no biometry is activated (i.e. only password or pin code). Did I miss something?

Reply:
You're right, but it would be still hidden by cozy-home hideSplashScreen intent (I had an issue where fingerprint view showed up before the LockScreen components). I will nonetheless try the scenario.

Reply:
Isn't the cozy-home intent called only on app's bootstrap? What about the scenario when the app's is resumed from background (the 1st time when `routes.lock` is not set yet)

Reply:
Yea I think I should add a hideSplashScreen() on first render just in case (having multiple hide/hiding if it's already hidden is not an issue)